authors/apps/articles/tests/test_article_views.py

```diff
@@ -22,6 +22,7 @@ def setUp(self):
         self.user2 = User.objects.create_user(
             username='user2', email='user2@mail.com', password='user2user2')
         self.user2.is_verified = True
+        self.user2.is_staff = True
```

Comment:
@BrianSerem, why are you making this test user staff, are only staff allowed to report articles?

Reply:
Only staff are allowed to view  all article reports, this user(user2) will act as staff when testing if a staff personnel can view all article reports